test/sagemaker_tests/pytorch/training/integration/local/test_single_machine_training.py

```diff
@@ -42,8 +42,8 @@ def test_mnist(docker_image, processor, instance_type, sagemaker_local_session,
 @pytest.mark.skip_py2_containers
 def test_fastai_mnist(docker_image, instance_type, py_version, sagemaker_local_session, tmpdir):
     _, image_framework_version = get_framework_and_version_from_tag(docker_image)
-    if Version(image_framework_version) == Version("1.9"):
-        pytest.skip("Fast AI is not supported on PT 1.9 DLC")
+    if Version(image_framework_version) == Version("1.9") or Version(image_framework_version) == Version("1.9.1"):
```

Comment:
lets add `in` to skip it for all pt1.9. Will it not be supported for further versions as well

Version(image_framework_version) in Version("1.9")

Reply:
Done

Reply:
@jeet4320 Seems `Version(image_framework_version) in Version("1.9")` is not working, could you take a look?